src/StructuralSolvers/StructuralSolvers.jl

```diff
@@ -216,7 +216,7 @@ By default `DEFAULT_LINEAR_SOLVER` is utilized.
 """
 function solve!(problem::AbstractStructuralAnalysis,
                 solver::Union{AbstractSolver,Nothing}=nothing,
-                linear_solve::LinearSolver=DEFAULT_LINEAR_SOLVER();
+                linear_solve::LinearSolver=DEFAULT_LINEAR_SOLVER;
```

Comment:
Here the solver is a type, probably that is why we get a method error in CI since is expecting an already instantiated solver object: `DEFAULT_LINEAR_SOLVER()` not the type. I suggest you to go through https://docs.julialang.org/en/v1/manual/methods/ for a full comprehension:  
```suggestion
                linear_solve::LinearSolver=DEFAULT_LINEAR_SOLVER();
```